test/e2e/util/kind/setup.go

```diff
@@ -20,25 +20,33 @@ import (
 	"bytes"
 	"flag"
 	"fmt"
-	"io"
-	"io/ioutil"
-	"os"
-	"os/exec"
-	"strings"
-
```

Comment:
I think we actually want this line to keep stdlib and external reps in separate sections? but also I don't super care.

Reply:
Changed a setting in VS Code.